src/core/scopes.cc

```diff
@@ -490,9 +491,14 @@ int32_t scopes::compile(const std::string& a_conf_dir_path)
                 WAFLZ_PERROR(m_err_msg, "missing customer id field");
                 return WAFLZ_STATUS_ERROR;
         }
+        if(!m_pb->has_account_type())
+        {
+                WAFLZ_PERROR(m_err_msg, "missing account_type field");
+        }
```

Comment:
You don't need this. This is not an error for existing configs. Remove this block